recipes/libsequence/build.sh

```diff
@@ -2,5 +2,7 @@
 
 CPPFLAGS="-I$PREFIX/include $CPPFLAGS" LDFLAGS="-L$PREFIX/lib $LDFLAGS" ./configure --prefix=$PREFIX
 make
+export LD_LIBRARY_PATH=-L$HOME/$PREFIX/lib
```

Comment:
@molpopgen You increased the wrong recipe. `libsequence` was never rebuild as far as I can tell. So these changes here do not have any effect. Please try to remove the libsequence changes. The build number from fwdpp can be at 0 as it is the first version of this recipe.